ironfish/src/blockchain/blockchain.ts

```diff
@@ -237,6 +237,7 @@ export class Blockchain {
 
     const result = await this.addBlock(genesis)
     Assert.isTrue(result.isAdded, `Could not seed genesis: ${result.reason || 'unknown'}`)
+    Assert.isTrue(result.isFork === false)
```

Comment:
```suggestion
    Assert.isFalse(result.isFork)
```

Reply:
isFork can be null, so explicit false check is needed.